packages/paste-core/components/modal/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@twilio-paste/modal",
+  "version": "0.0.0",
+  "category": "overlay",
+  "status": "beta",
+  "description": "An alert notifies a user to high-priority or time-sensitive information.",
```

Comment:
I think this is the Alert package description.

Reply:
It is, good spot

Reply:
Updated